app/src/main/res/layout/list_item.xml

```diff
@@ -0,0 +1,62 @@
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/clListItem"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_margin="16dp"
+    android:background="#80000000">
+
+    <TextView
+        android:id="@+id/textView_temperature"
```

Comment:
please check proper naming convention for resource ids :)

Reply:
Okay noted!

Reply:
This file is already removed in my App